cmd/convox/templates/init/rails/docker-compose.yml

```diff
@@ -6,3 +6,5 @@ web:
   ports:
     - 80:4000
     - 443:4001
+  volumes:
```

Comment:
Has anyone done any performance testing (even just anecdotal) with volumes on Docker for Mac? The existing solutions with volumes had real problems around performance that made them basically unusable. Are those problems gone?